Sources/Datadog/Logger.swift

```diff
@@ -70,55 +76,61 @@ public class Logger {
     /// Sends a DEBUG log message.
     /// - Parameters:
     ///   - message: the message to be logged
+    ///   - error: `Error` instance to be logged with its properties
```

Comment:
structs and enums are logged with their properties
class support may come with #304